WordPressAuthenticator/Signin/AppleAuthenticator.swift

```diff
@@ -267,3 +213,80 @@ extension AppleAuthenticator {
         ASAuthorizationAppleIDProvider().getCredentialState(forUserID: userID, completion: completion)
     }
 }
+
+// This needs to be internal, at this point in time, to allow testing.
+//
+// Notice that none of this code was previously tested. A small encapsulation breach like this is
+// worth the testability we gain from it.
+extension AppleAuthenticator {
+
+    func showSignupEpilogue(for credentials: AuthenticatorCredentials) {
+        guard let navigationController = showFromViewController?.navigationController else {
+            fatalError()
+        }
+
+        let service = loginFields.meta.appleUser.map {
+            SocialService.apple(user: $0)
+        }
+
+        authenticationDelegate.presentSignupEpilogue(in: navigationController, for: credentials, service: service)
+    }
+
+    func createWordPressComUser(appleUserId: String, email: String, name: String, token: String) {
+        tracker.set(flow: .signupWithApple)
+        tracker.track(step: .start) {
+            track(.createAccountInitiated)
+        }
+
+        SVProgressHUD.show(
+            withStatus: NSLocalizedString(
+                "Continuing with Apple",
+                comment: "Shown while logging in with Apple and the app waits for the site creation process to complete."
+            )
+        )
+
+        updateLoginFields(email: email, fullName: name, token: token)
+
+        signupService.createWPComUserWithApple(
```

Comment:
2/2 changes from the code deleted above: use the `signupService` property instead of instantiating ad hoc.